<?xml version="1.0" encoding="UTF-8"?>

<launch>
<!--GPS-->
	<include file="$(find ublox_gps)/launch/ublox_device.launch">
	    <arg name="node_name" value="ublox_gps"/>
	    <arg name="param_file_name" value="zed-f9p"/>
	    <arg name="output" value="screen"/>
	    <arg name="respawn" value="true"/>
	    <arg name="respawn_delay" value="30"/>
	    <arg name="clear_params" value="false"/>
	</include>

 <node pkg="ntrip_ros" type="ntripclient.py" name="ntrip_ros" output="screen">
 	 <param name="rtcm_topic" value="/send_rtcm"/>
 	 <param name="ntrip_server" value="vrs3.ngii.go.kr:2101"/>
 	 <param name="ntrip_user" value="ss4001aa"/>
 	 <param name="ntrip_pass" value="ngii"/>
 	 <param name="ntrip_stream" value="VRS-RTCM31"/>
 	 <param name="nmea_gga" value="$GPGGA,024539.902,3723.004,N,12639.255,E,1,12,1.0,0.0,M,0.0,M,,*60"/>
  </node>
</launch>
